src/it/scala/net/snowflake/spark/snowflake/PushdownEnhancement01.scala

```diff
@@ -105,7 +105,8 @@ class PushdownEnhancement01 extends IntegrationSuiteBase {
          |FROM ( SELECT * FROM ( $test_table_length ) AS
          |"SF_CONNECTOR_QUERY_ALIAS" ) AS "SUBQUERY_0" """.stripMargin,
       result,
-      expectedResult
+      expectedResult,
+      testPushdownOff = false
```

Comment:
To Reviewers:
If use_copy_unload = true, the test fails. This difference comes from the limitation for COPY UNLOAD NULL value processing. It can be ignored.

1. When reading with COPY UNLOAD, empty string will be read as NULL. (With Arrow, it is ready as "").
2. This test case test: length(""), length("abc"), length(null).
3. The expected result is (0, 3, NULL).
4. With COPY UNLOAD, the data is read as (null, "abc", null). So the final result is (null, 3, null).